code/WorkInProgress/warcrimes.dm

```diff
@@ -47,7 +46,7 @@ TYPEINFO(/area/diner/juicer_trader)
 			boutput(user, "You staple the [src] together to create a mighty thongarang.")
 			name = "thongarang"
 			icon_state = "thongarang"
-			throwforce = 5
+			throwforce = 0
```

Comment:
```suggestion
			throwforce = 1
```

Reply:
I dunno, I find this to be a funny item that people can just spam throw at someone else. Making it do actual damage - even if its 1 - while its spammed can do around 30 brute in a few seconds, which I find a bit too much since people would probably wanna use this as a joke without actually hurting the person. 
The -1 melee damage reduction for jumpsuits doesnt seem to negate it.